tests/posix/message_passing/src/main.c

```diff
@@ -40,13 +40,11 @@ static void *sender_thread(void *p1)
 	zassert_false(mq_timedsend(mqd, send_data, MESSAGE_SIZE, 0, &curtime),
 		      "Not able to send message in timer");
 	usleep(USEC_PER_MSEC);
-	zassert_false(mq_close(mqd),
-		      "unable to close message queue descriptor.");
+	zassert_false(mq_close(mqd), "unable to close message queue descriptor.");
```

Comment:
If CI doesn't complain, I think my preference would be to omit unnecessary formatting changes.